client/state/stats/lists/reducer.js

```diff
@@ -84,6 +85,27 @@ export const items = withSchemaValidation( itemSchema, ( state = {}, action ) =>
 					},
 				},
 			};
+		}
+
+		case ALL_SITES_STATS_RECEIVE: {
+			const { query, data } = action;
+			const newState = { ...state };
+
+			for ( const siteId in data.stats ) {
+				const siteData = data.stats[ siteId ];
```

Comment:
`siteData` is used only once; I'd suggest removing the constant and inlining `data.stats[ siteId ]`.